src/components/settings/Calendars.js

```diff
@@ -203,14 +219,61 @@ export default class Calendars extends AddDeleteSetting {
     this.setState({ hexColor })
   }
 
+  handleFieInputChange = event => {
+    const { verifyNewCalendar } = this.props
+
+    console.log('selected files', event.target.files[0])
+    if (event.target.files[0]) {
+      let file = event.target.files[0]
+      let fileExt = file.name.split('.').pop()
+
+      if (fileExt.toLowerCase() !== 'ics') {
+        alert('Invalid file type')
+        return
+      }
+
+      let fileReader = new FileReader()
+      fileReader.onloadend = e => {
+        console.log('read file complete')
+
+        let calendar = {
+          uid: uuid(),
+          type: 'ics',
+          name: this.state.calendarName,
+          hexColor: this.state.hexColor,
+          mode: 'read-only',
```

Comment:
This shouldn't be `read-only`.  We need to upload the ics file to gaia and then the user can update the events on there. Can you just remove that please?

This wasn't clear to me and is not clear from the issue, sorry.

Reply:
Ok.